00-HTML-CSS-basics/The Basics of HTML/Questions.txt

```diff
@@ -1,5 +1,11 @@
+General:
+
+I accidentally pushed a commint to the master branch.
+Can I remove it somehow with rebase?
+
+
 HTML:
 
 1-5)
 Should I upload a resource folder containing the image, video and sound files?
-If so, how should I type the source of each of them?
\ No newline at end of file
+If so, how should I type the source of each of them?
```

Comment:
In case you need a resource folder, you need to use relative path to link them with your html file. [Here](http://www.coffeecup.com/help/articles/absolute-vs-relative-pathslinks/) is an article that maybe can be helpful.